mesh/v1alpha1/config.proto

```diff
@@ -42,6 +42,18 @@ enum AuthenticationPolicy {
   INHERIT = 1000;
 }
 
+// LightstepCollector defines configuration for a LightStep Satellite pool.
+message LightstepCollector {
+  // Address of the LightStep Satellite pool.
+  string address = 1;
+
+  // The LightStep access token.
+  string access_token = 2;
+
+  // True if a secure connection should be used when communicating with the pool.
+  bool secure = 3;
```

Comment:
Is there any prior art for this? My only thought is that maybe this should default to secure and allow overloading to insecure.

Reply:
Great question. Within this file AuthenticationPolicy defaults to "none" rather than mutual_tls which is something but I agree that secure should probably be the default. 

I'm thinking we should update this to `insecure`. Thoughts?

Reply:
That sounds good to me but I would be interested to hear offical feedback on that design. Looking forward to review.

Reply:
On second thought, Envoy defaults to insecure. I'm going to leave it. I'll be happy to do whatever they end up suggesting.